pkg/subctl/benchmark/throughput.go

```diff
@@ -0,0 +1,182 @@
+package benchmark
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/submariner-io/shipyard/test/e2e/framework"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("[throughput] Test throughput between two Clusters", func() {
```

Comment:
Oh no no no, please don't use the ginkgo framework for this.

It is ok that we use the Framework objects if you need (to setup pods, do anything across clusters). But I'd avoid the Describe / When, etc... just use the "Framework" stuff, you may need to call the framework.

This will allow us to do proper reporting, like the bandwidth, etc... not just PASSED/FAILED. Ginkgo/Gomega are for unit testing    PASSED/FAILED testing... here we want is a report with the characteristics of the network.

Reply:
So @mangelajo. If we use gingko framework, we will know the various steps of spawning a pod and if there is any failure in any step we will know it easily. Also the idea was that we will just dump the output of iperf3 client/server for user to make interpretations and the tool will NOT parse the iperf3 output. 

Do you think its fine not to show the various steps like spawning a pod in a particular process, trying to connect from iperf3 client to server etc?

Also, do you think the tool should parse the output and only display some summary results?

Reply:
.By function seems to be safe called outside of ginkgo framework... just goes to stdout.